util/sst_file_manager_impl.cc

```diff
@@ -106,13 +106,13 @@ bool SstFileManagerImpl::IsMaxAllowedSpaceReachedIncludingCompactions() {
          max_allowed_space_;
 }
 
-bool SstFileManagerImpl::EnoughRoomForCompaction(Compaction* c) {
+bool SstFileManagerImpl::EnoughRoomForCompaction(const std::vector<CompactionInputFiles> inputs) {
```

Comment:
can it be passed as reference-to-const-vector? Same for the `DBImpl` function

Reply:
done